doc/api/n-api.md

```diff
@@ -3207,13 +3207,16 @@ added: v8.0.0
 ```C
 NAPI_EXTERN
 napi_status napi_create_async_work(napi_env env,
+                                   const char* async_resource_name,
```

Comment:
Yup. At this point I don’t think we can do much beyond encouraging users to do proper namespacing. :)